### ISSUE TITLE
New option to redirect after entry update

### ISSUE THREAD

**Issue by rafaehlers** [gravityview/GravityView#698]
- Original entry (default)
- Multiple Entries screen
- Other URL (text input)

Note: Need to move `process_save` out of `GravityView_Edit_Entry_Render::init` to it's own calling method, hooked before render

![gv-idea](https://cloud.githubusercontent.com/assets/1222472/15238115/18b470a8-18ab-11e6-9039-e1be2b466020.jpg)

**Comment by zackkatz** [comment]
@soulseekah it should have a drop down and the Custom URL option should then show another text box with an URL

**Comment by soulseekah** [comment]
I can work on it once the UI is in place. I'd add it myself, but the "then show another text box" is not my area of expertise, sorry :)

Call the setting `edit_redirect`, set it to empty (0 or '') for regular behavior, '1' to return to multiple view, and any other value will do a 302 redirect.

Ping me with the branch name. Should be easy to pull off once the setting is in place.

**Comment by soulseekah** [comment]
The largest issue here is that `process_save` is called after output has started. This means that we can't issue any redirect headers at that point.

We either need to process the form earlier which I don't feel too comfortable with doing, too easy to break.

The alternative is to output a location.redirect script and a fallback link. PoC in the `feature/edit-redirect` branch.

The setting is called `edit_redirect`. Set it to "" (empty string) to keep default behavior of staying on the page, "0" to return to entry, "1" to return to directory, anything else to redirect to wherever anything else is.

**Comment by soulseekah** [comment]
Having been merged into develop and 2.4 fast approaching, this still needs the UI to be completed. At least it's not showing up for me at all:

![image](https://user-images.githubusercontent.com/685880/49167209-ff850200-f345-11e8-8c7a-554ba2bef332.png)

Tests are fine. This is urgent borked functionality in 2.0.
